### PR TITLE
Vnip2 sgrulelocal

### PR DESCRIPTION
**Pull request by SunithaGudisagarIBM1**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

**Comment by ujjwal-ibm** [comment]
rebase the PR

File: ibm/service/vpc/resource_ibm_is_security_group.go [hunk 1]
```diff
@@ -289,6 +289,16 @@ func resourceIBMISSecurityGroupRead(d *schema.ResourceData, meta interface{}) er
 							}
 						}
 					}
+					local, ok := rule.Local.(*vpcv1.SecurityGroupRuleLocal)
```
Comment (ujjwal-ibm) [inline ibm/service/vpc/resource_ibm_is_security_group.go:292]: test cases updation required

**Comment by SunithaGudisagarIBM1** [comment]
we will be using https://github.com/ibm-vpc/terraform-provider-ibm/pull/301 PR which has the rebased changes and the updated test cases which has been tested..

**Comment by SunithaGudisagarIBM1** [comment]
we will be using https://github.com/ibm-vpc/terraform-provider-ibm/pull/301 PR which has the rebased changes and the updated test cases which has been tested..

**Pull request closed without merging**